### PR TITLE
Rename HealUnitsCrateAction to HealActorsCrateAction

### LINKED ISSUE #17912: HealUnitsCrateAction doesn't heal (just) units
The current implementation heals all actors, including buildings. The trait should either be fixed or renamed to reflect what it actually does.

IMO the best solution would be to rename it to `HealActorsCrateAction` and add a `TargetTypes` property that can be used to control which actors are affected.

### PR DESCRIPTION
**Pull request by abcdefg30**
Closes #17912.

### CODE REVIEW

**Comment by Mailaender** [comment]
Why not set `TargetTypes` so that they only affect units?

**Comment by pchote** [comment]
This will need a rebase + update rule move after #18038.

File: OpenRA.Mods.Common/Traits/Crates/HealActorsCrateAction.cs [hunk 1]
```diff
@@ -10,25 +10,34 @@
...
 using OpenRA.Traits;
 
 namespace OpenRA.Mods.Common.Traits
 {
 	[Desc("Heals all actors that belong to the owner of the collector.")]
-	class HealUnitsCrateActionInfo : CrateActionInfo
+	class HealActorsCrateActionInfo : CrateActionInfo
 	{
-		public override object Create(ActorInitializer init) { return new HealUnitsCrateAction(init.Self, this); }
+		[Desc("The target type(s) of the actors this crate action will heal. Left empty to heal all actors.")]
```
Comment (pchote) [inline OpenRA.Mods.Common/Traits/Crates/HealActorsCrateAction.cs]: "Left" &rarr; "Leave"

**Review by pchote — COMMENTED** [review]
LGTM otherwise

**Comment by abcdefg30** [comment]
Updated.

**Comment by Mailaender** [comment]
Still the "bug" that heal unit crates heal all buildings remains in all shipped mods.

**Comment by abcdefg30** [comment]
We didn't consider that as bug: https://github.com/OpenRA/OpenRA/issues/17912#issuecomment-610961604

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/61334738c4e6066399fd5f5c4287dede504836b9...0d9e3038ca90d60c88704e9f4777afb923e2c9b1

**Pull request merged**